Formula/citus.rb

```diff
@@ -19,22 +19,24 @@ class Citus < Formula
   depends_on "readline"
   depends_on "zstd"
 
+  uses_from_macos "curl"
+
   def install
     ENV["PG_CONFIG"] = Formula["postgresql"].opt_bin/"pg_config"
 
     system "./configure"
 
-    # workaround for https://github.com/Homebrew/homebrew/issues/49948
+    # workaround for https://github.com/Homebrew/legacy-homebrew/issues/49948
     system "make", "libpq=-L#{Formula["postgresql"].opt_lib} -lpq"
 
     # Use stage directory to prevent installing to pg_config-defined dirs,
     # which would not be within this package's Cellar.
     mkdir "stage"
     system "make", "install", "DESTDIR=#{buildpath}/stage"
 
-    bin.install Dir["stage/**/bin/*"]
-    lib.install Dir["stage/**/lib/*"]
-    include.install Dir["stage/**/include/*"]
-    (share/"postgresql/extension").install Dir["stage/**/share/postgresql/extension/*"]
+    path = File.join("stage", HOMEBREW_PREFIX)
+    lib.install Dir["#{path}/lib/*"]
+    include.install Dir["#{path}/include/*"]
+    (share/"postgresql/extension").install Dir["#{path}/share/postgresql/extension/*"]
```

Comment:
It's kinda weird that the previous globs didn't work on Linux though.
```suggestion
    lib.install (buildpath/path/"lib").children
    include.install (buildpath/path/"include").children
    (share/"postgresql/extension").install (buildpath/path/"share/postgresql/extension").children
```